sdccc/src/test/java/com/draeger/medical/sdccc/tests/glue/direct/DirectWSDLServiceDescriptionsTestTest.java

```diff
@@ -213,16 +216,119 @@ public void testRequirement13BadImportElementPresent() throws Exception {
         assertThrows(AssertionError.class, testClass::testRequirement13);
     }
 
+    /**
+     * Tests whether the test passes when the Device provides a DescriptionEventService.
+     *
+     * @throws Exception on any exception
+     */
+    @Test
+    public void testRequirement813Good() throws Exception {
+
+        final HostingServiceProxy hostingServiceProxy = mock(HostingServiceProxy.class);
+        when(testClient.getHostingServiceProxy()).thenReturn(hostingServiceProxy);
+        final HostedServiceProxy descriptionEventServiceProxy =
+                mock(HostedServiceProxy.class, Answers.RETURNS_DEEP_STUBS);
+        when(hostingServiceProxy.getHostedServices())
+                .thenReturn(Map.of("DescriptionEventService", descriptionEventServiceProxy));
```

Comment:
Why don't you use the WsdlConstants here as well?

Reply:
Done.